src/opendr/planning/end_to_end_planning/e2e_planning_learner.py

```diff
@@ -21,31 +21,37 @@
 from stable_baselines3 import PPO
 from stable_baselines3.common.monitor import Monitor
 from stable_baselines3.common.vec_env import DummyVecEnv
-from stable_baselines3.common.results_plotter import load_results, ts2xy
+from stable_baselines3.common.callbacks import CheckpointCallback
 
 from opendr.engine.learners import LearnerRL
 from opendr.engine.constants import OPENDR_SERVER_URL
 
-__all__ = ["rospy", ]
+# __all__ = ["rospy", ]
 
 
 class EndToEndPlanningRLLearner(LearnerRL):
-    def __init__(self, env, lr=3e-4, n_steps=1024, iters=int(5e4), batch_size=64, checkpoint_after_iter=500,
+    def __init__(self, env=None, lr=3e-4, n_steps=1024, iters=int(1e5), batch_size=64, checkpoint_after_iter=500,
```

Comment:
Changes to default values should be updated accordingly in the documentation file as well (`end-to-end-planning.md`)

Reply:
updated